docs/source/search-runs.rst

```diff
@@ -29,17 +29,41 @@ Example Expressions
 
```

Comment:
@apurva-koti Nice work! Can we also add examples for searching by attribute columns matching the row headers (i.e. relates to the support you're adding to match the column names properly)?

Reply:
@dbczumar there will be a couple more changes to this page to support new features (including that), but I don't think we should merge them before those features exist

Reply:
@apurva-koti Got it! Can we file an internal JIRA to track updating these docs when we add the new capabilities?

Reply:
DMed earlier :)